Source/iOS/OIDExternalUserAgentIOSCustomBrowser.h

```diff
@@ -20,6 +20,8 @@
 
 #import "OIDExternalUserAgent.h"
 
+#if !TARGET_OS_MACCATALYST
```

Comment:
For discussion (am not asking for changes here):

One implication of this, is that anyone who uses OIDExternalUserAgentIOSCustomerBrowser will get missing symbols when they build for Catalyst.  Maybe that's OK, since they can't use it in any case.

Another approach would be to implement stub methods for Catalyst that just assert.

The benefit of the assert is that it can explain what to do ("you must use Catalyst specific user-agents on Catalyst"). The benefit of the missing symbols is that it's compile time rather than runtime, although the compiler message may confuse.

Reply:
Personally I prefer getting an error at compile time rather than runtime, but the assert approach has a valid point for the clarity of the message. Although the current approach doesn't give a clear message to the user, the user agents names give a good indication and suggest that some are for iOS and other for Catalyst, having that in their class name.

I'm not sure what's the best practice for this use case... but I'm open to implement what is best for the user.